packages/vuetify/src/components/VTreeview/VTreeviewNode.ts

```diff
@@ -334,13 +373,80 @@ const VTreeviewNode = baseMixins.extend<options>().extend({
 
       const children = [this.children.map(c => this.genChild(c, this.disabled))]
 
-      return this.$createElement('div', {
+      return this.$createElement('div', { // ul
+        attrs: {
+          'aria-labeledby': this.htmlId,
+          id: this.ariaControls,
+          tabindex: -1,
+          role: 'group',
+        },
         staticClass: 'v-treeview-node__children',
       }, children)
     },
     genTransition () {
       return this.$createElement(VExpandTransition, [this.genChildrenWrapper()])
     },
+    handleFocus (event: Event): void {
+      if (event.target) {
+        this.treeview.updateFocus(this.key, event.type === 'focus')
+      }
+    },
+    handkeKey (event: KeyboardEvent): void {
+      const moveDown = (event: KeyboardEvent) => {
+        this.treeview.keyMoveDown(this.key)
+        event.stopPropagation()
+      }
+      const moveUp = (event: KeyboardEvent) => {
+        this.treeview.keyMoveUp(this.key)
+        event.stopPropagation()
+      }
+      const moveLeft = (event: KeyboardEvent) => {
+        this.treeview.keyMoveLeft(this.key)
+        event.stopPropagation()
+        this.updateScroll(event.currentTarget)
+      }
+      const moveRight = (event: KeyboardEvent) => {
+        if (this.isOpen) {
+          this.treeview.keyMoveRight(this.key)
+        } else {
+          this.checkChildren().then(() => {
+            if (this.hasChildren) {
+              this.open()
+              this.$nextTick(function () {
+                this.updateScroll(event.currentTarget)
+              })
+            }
+          })
+        }
+        event.stopPropagation()
+      }
+      const toggleOpen = (event: KeyboardEvent) => {
+        if (this.hasChildren) {
+          this.checkChildren().then(this.open)
+          this.$nextTick(function () {
+            this.updateScroll(event.currentTarget)
+          })
+        }
+      }
+
+      const keyCode = event.keyCode
+      if ([keyCodes.down].includes(keyCode)) return moveDown.call(this, event)
```

Comment:
- Includes on a one-element array is redundant
- Arrow functions use lexical `this`, they can be called directly.